Wireframe/index.css

```diff
@@ -1,62 +1,69 @@
 
 body {
```

Comment:
I notice there is a thin gap on the left (and right) sides of the body where the background colors do not extend to the edge of the page -- why might this be and how can we fix it?

Reply:
I set the margin on all sides to zero, this resolved the issue.